private-templates-service/client/src/components/Welcome/Welcome.tsx

```diff
@@ -37,9 +39,9 @@ const WelcomeContent = (props: WelcomeProps): ReactElement => {
       </Button>
     </React.Fragment>
   );
-}
+};
```

Comment:
Don't need the ;
I wonder if your linter config is different from others in your team or something? These look auto added